apps/src/studio/studio.js

```diff
@@ -2132,6 +2228,11 @@ var ANIM_AFTER_NUM_NORMAL_FRAMES = 8;
 // reset them to face south.
 var IDLE_TICKS_BEFORE_FACE_SOUTH = 4;
 
+Studio.itemGetDirectionFrame = function(item) {
+  return item.dir == Direction.NONE ? 0 : frameDirTableWalking[item.dir];
```

Comment:
Might be better to have frameDirTableWalking live in and get exported from src/studio/constants.js, allowing us to have getDirectionFrame live in Item.js. We could also add an entry:
```
frameDirTableWalking[Direction.NONE]      = 0;
```